zerver/views/registration.py

```diff
@@ -295,13 +296,21 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
             # create an unusable password).
             password = None
 
+        # In the case of
+        browser_locale = request.META.get("HTTP_ACCEPT_LANGUAGE")
+        default_language = browser_locale.split(",")[0] if browser_locale is not None else "en"
```

Comment:
I think this extracting the language from browser_locale can be separated into a diffrent function. I am guessing this is not the first place we are doing this right?

Reply:
This was the first place where we were doing it, but it will be better to move it to a different function. I have done that in the recent commits.